### PR TITLE
Create CI pipeline 

### PR DESCRIPTION
**Pull request by MaksimSurmach**
Add a Drone pipeline for build publish docker image automatically.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+steps:
+- name: build 
+  image: plugins/docker
+  settings:
+    username: 
+      from_secret: docker_username
+    password:
+      from_secret: docker_password
+    repo: maxsurm/prospero
```
Comment (kinkard) [inline .drone.yml:14]: From fork? :)

Comment (MaksimSurmach) [reply .drone.yml:14]: just insert your credentials to Drone secrets :)  
Or we can try build and publish docker image in Github Action, and deploy via Drone CI

Comment (kinkard) [reply .drone.yml:14]: I've set up GH Actions to build docker image and publish  it to the [Docker Hub](https://hub.docker.com/repository/docker/kinkard/prospero)  - https://github.com/kinkard/prospero/blob/main/.github/workflows/push_docker_image.yml

Comment (MaksimSurmach) [reply .drone.yml:14]: Okay, maybe you're need a help with autodeploy?
And I think this PR not relevant

Comment (kinkard) [reply .drone.yml:14]: Decently yes

**Pull request closed without merging**